### PR TITLE
console: raw sql section bug fixes (close #3752)

### LINKED ISSUE #3752: Track this option is checked automatically for commented out SQL
If you have a SQL like this 
-- CREATE VIEW e_space_note_view AS
event though it is commented out, the track this gets checked.

### PR DESCRIPTION
**Pull request by abhi40308**
Earlier, when writing commented SQL in Raw SQL section, commented SQL code also got tracked, due to which `track this` and `this is a migration` checkboxes behaved in unwanted ways. This pr fixes that bug.

### Description
fix #3752 

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console
- [x] CLI

### Steps to test and verify
Go to Raw SQL section and type commented SQL code in the text editor.

### CODE REVIEW

**Review by soorajshankar — APPROVED** [review]
LGTM. Tested with and without SQL comments and the checkbox works as expected.

**Review by beerose — APPROVED** [review]
Console

File: console/src/components/Services/Data/RawSQL/RawSQL.js [hunk 1]
```diff
@@ -153,16 +153,17 @@ const RawSQL = ({
     const handleSQLChange = val => {
       onChangeSQLText(val);
       dispatch({ type: SET_SQL, data: val });
+      const cleanSql = removeCommentsSQL(val);
```
Comment (abhi40308) [inline console/src/components/Services/Data/RawSQL/RawSQL.js:156]: TODO: Changes after multiple datasources

**Comment by beerose** [comment]
Closed in favor of https://github.com/hasura/graphql-engine-mono/pull/328

**Pull request closed without merging**